conf/app.routes

```diff
@@ -2,7 +2,10 @@
 
 ->         /govuk-frontend          govuk.Routes
 ->         /hmrc-frontend           hmrcfrontend.Routes
+GET        /assets/*file            controllers.Assets.versioned(path = "/public", file: Asset)
+
 GET        /hello-world             uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.HelloWorldController.helloWorld
```

Comment:
remove this endpoint

Reply:
I cannot remove it!
When I tried, I got an exception from: views/html/Head.template.scala:16